### PR TITLE
v0.59.0 added support for arm

### PR DESCRIPTION
**Pull request by nmcc1212**
update to match https://github.com/MicrosoftDocs/windows-uwp/pull/3899

### CODE REVIEW

**Comment by PRMerger3** [comment]
@nmcc1212 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@mattwojo

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: hub/powertoys/index.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 ## Processor support
 
 - **x64**: Supported
-- **Arm**: In development (see [issue #490](https://github.com/microsoft/PowerToys/issues/490))
-
+- **Arm**: Supported
```
Comment (Jay-o-Way) [inline hub/powertoys/index.md:21]:
```suggestion
- **ARM**: Supported
```

**Review by Jay-o-Way — COMMENTED** [review]
Might as well make it uppercase

**Comment by PRMerger17** [comment]
@nmcc1212 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mattwojo** [comment]
> Might as well make it uppercase

We have an agreement with the ARM organization to use "Arm" in our docs.

**Comment by mattwojo** [comment]
#sign-off

**Pull request merged**